uint32 inPathId
float64 atDistance
---
float64[] ret
